README.md

```diff
@@ -98,6 +102,23 @@ Determines if the component owners should be added to the pull request as assign
 
 Determines pull request reviews should be requested from component owners.
 
+## Using own access token
+
+If you want to use this action to assign reviews for teams, then `github.token` would not be sufficient and you need to generate a new one. 
+
+1. Just follow [Github's instructions](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/managing-your-personal-access-tokens#creating-a-fine-grained-personal-access-token) and add a new token with the following set of permissions:
+
+![perm](https://github.com/volas/component-owners/assets/4064061/474022d1-820a-48c6-8100-a8553584e961)
```

Comment:
If this PR gets merged this should point to the upstream repo or have a link independent of the repo name

Reply:
Replaced it with a markdown table.

Also removed Discussions prem as it was unnecessary, I think.